vaadin-form-layout-flow-parent/vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java

```diff
@@ -922,9 +921,13 @@ public void setColumnWidth(float columnWidth, Unit unit) {
     /**
      * Gets the width of columns that is used when
      * {@link #setAutoResponsive(boolean)} is enabled.
+     * <p>
+     * When the column width is {@code null}, the web component defaults to
+     * {@code 12em} or uses the value of {@code --vaadin-field-default-width`}
```

Comment:
nit: extra backtick.

```suggestion
     * {@code 12em} or uses the value of {@code --vaadin-field-default-width}
```